app/(editor)/editor/page.tsx

```diff
@@ -67,7 +67,7 @@ async function getStoryData({ path }: PageProps['searchParams']): Promise<ISbRes
   const slug = path.replace(/\/$/, '') || 'home'; // Remove trailing slash or if no slash, use home.
 
   try {
-    const story: ISbResult = await storyblokApi.get(`cdn/stories/${slug}`, sbParams);
+    const story: ISbResult = await storyblokApi.get(`cdn/stories/${slug}`, sbParams, { cache: 'no-store' });
```

Comment:
Storyblok uses the Fetch API under the hood and has now allowed for us to pass optional parameters through to it. 

see: https://nextjs.org/docs/app/building-your-application/caching#data-cache